design/decisions/why-sql/index.md

```diff
@@ -1,16 +1,194 @@
+with this form for database system and we will have users that will
+require access to the raw data (see the user types definition here [link
+to the user types]()). This means that we are not looking at a system
+like MongoDB and Redis.
+
+As we are planning to use container technology to run the database it is
+not as important which operating systems the database will run on.
+Having said that, of the three systems that we are looking at MySQL, and
+PostgreSQL will run on multiple operating systems (e.g. Linux, Mac OS,
+and Windows), and SQLite is a classic serverless application.
+
+A [side-by-side
+comparison](https://db-engines.com/en/system/MySQL%3BPostgreSQL%3BSQLite)
+on [DB-engines](https://db-engines.com) was used to compile some of the
+comparison below.
+
+## [MySQL](www.mysql.com)
+
+First released in 1995 it is maintained by Oracle Corp and is (at the
+time of writing) on version 8.0.31. It is an open source platform with
+the option to deploy either as a local server solution or cloud based.
+The implementation languages are C and C++ and it runs of a variety of
+operating systems. The system allows access through standard
+technologies (ADO.NET, JDBC, ODBC, and native APIs).
+
+**Benefits**
+
+-   At present the second most popular database both open source and
+    overall with good support and a large community.
+-   Traditional database system with a recognisable format, should be
+    easy to manipulate and work with for the advanced seedcase user.
+-   Support for both xml and json formats, both reading and writing.
+-   There are a number of ways for MySQL to interact with Apache Parquet
+    files.
+
+**Drawbacks**
+
+-   There is always a risk that an open source community will break
+    apart and leave a product unsupported, but the risk here looks
+    minimal.
+-   MySQL is run by Oracle which is a commercial entity. There is always
+    a risk that the company decides to reverse the open source concept
+    and go move to a solution with a free light version and full payable
+    version. In this case it is very unlikely as the software is very
+    well established and the user community quite large.
+-   There is currently no option in MySQL to store data in a columnar
+    table.
+
+## [PostgreSQL](www.postgresql.org)
+
+First released in 1989 from UC Berkeley, it is maintained by the
+PostgreSQL Development Group and is (at the time of writing) on version
+15.1. It is an open source platform with the option to deploy either as
+a local server solution or cloud based. The implementation language is C
+and it runs of a variety of operating systems. The system allows access
+through standard technologies (ADO.NET, JDBC, ODBC, a native C library,
+and streaming APIs).
+
+**Benefits**
+
+-   At present the fourth most popular database overall and the second
+    most popular open source db. There is a thriving community with a
+    lot of engage users delivering support.
```

Comment:
```suggestion
    lot of engaging users delivering support.
```

Reply:
I was going for engaged (missing a 'd'), how do I suggest a change to a suggested change?

Reply:
You can edit it directly, that will cancel my suggestion :stuck_out_tongue: